rclcpp_examples/CMakeLists.txt

```diff
@@ -68,12 +68,21 @@ if(BUILD_TESTING)
     parameter_events
     set_and_get_parameters_async
     set_and_get_parameters
-    "talker:listener"
+    "talker:listener")
+  set(fastrtps_excluded_example_tests
     "add_two_ints_server:add_two_ints_client"
     "add_two_ints_server:add_two_ints_client_async")
 
   macro(tests)
-    foreach(example_test ${example_tests})
+    set(example_tests_to_test ${example_tests})
+    # TODO(wjwwood): recombine with example_tests this when fastrtps supports wait_for_service.
+    if("${rmw_implementation} " STREQUAL "rmw_fastrtps_cpp ")
```

Comment:
Since we are using CMake 3.5 now please avoid the trailing spaces as well as the quotes and explicit variable expansion if the variable is know to exist:

    if(rmw_implementation STREQUAL "rmw_fastrtps_cpp")